### PR TITLE
test: Move some Services test cases to separate suites

### PR DESCRIPTION
**Pull request by brb**
See commit msgs.

### CODE REVIEW

**Comment by brb** [comment]
/test-1.23-net-next

**Comment by brb** [comment]
/test-1.23-net-next

**Comment by brb** [comment]
/test-1.23-net-next

**Comment by brb** [comment]
/test-1.23-net-next

**Comment by brb** [comment]
/test-1.23-net-next

**Comment by brb** [comment]
/test-1.23-net-next

**Comment by brb** [comment]
/test-1.23-net-next

**Comment by brb** [comment]
/test-1.23-net-next

**Review by ldelossa — APPROVED** [review]
PR of the year :rocket:

File: test/k8sT/Policies.go [hunk 1]
```diff
@@ -2340,6 +2340,120 @@ var _ = SkipDescribeIf(func() bool {
 				"%q Clusterwide Policy cannot be deleted", ingressDenyAllPolicy)
 		})
 	})
+
+	//TODO: Check service with IPV6
```
Comment (christarazi) [inline test/k8sT/Policies.go:2344]: Should we file an issue here given that we've had previous discussions to convert TODOs in code to issues?

**Review by christarazi — APPROVED** [review]
Thanks!

File: test/k8sT/Policies.go (hunk 1 above)
Comment (brb) [reply test/k8sT/Policies.go:2344]: Good point! I'm going to file the issue to move the whole suite to a unit test :-)

**Pull request merged**